### PR TITLE
Fix invalid conversion host resource type

### PR DESCRIPTION
**Pull request by agrare**
Fix an invalid polymorphic resource_type from
https://github.com/ManageIQ/manageiq-schema/pull/281 which resulted in
the type being "AddConversionHostIdToMiqRequestTasks::Host" instead of
"Host".

### CODE REVIEW

File: db/migrate/20181002192054_fix_conversion_host_resource_type.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class FixConversionHostResourceType < ActiveRecord::Migration[5.0]
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
```
Comment (bdunne) [inline db/migrate/20181002192054_fix_conversion_host_resource_type.rb]: You'll also need an `include ActiveRecord::IdRegions`

File: db/migrate/20181002192054_fix_conversion_host_resource_type.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  end
+
+  def up
+    say_with_time("Converting invalid resource_type to Host") do
+      ConversionHost.in_my_region
```
Comment (bdunne) [inline db/migrate/20181002192054_fix_conversion_host_resource_type.rb]: I think this should only be scoped if the other migration is scoped.

Comment (agrare) [reply db/migrate/20181002192054_fix_conversion_host_resource_type.rb]: Want to change the other migration to be scoped in this PR?

Comment (bdunne) [reply db/migrate/20181002192054_fix_conversion_host_resource_type.rb]: Yes, please

File: spec/migrations/20181002192054_fix_conversion_host_resource_type_spec.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+  let(:host_stub)            { migration_stub(:Host) }
+  let(:vm_stub)              { migration_stub(:Vm) }
+  let(:conversion_host)      { conversion_host_stub.create! }
+  let(:host)                 { host_stub.create! }
+  let(:vm)                   { vm_stub.create! }
+
+  migration_context :up do
+    it "Updates invalid resource_types" do
+      conversion_host.update_attributes(
```
Comment (bdunne) [inline spec/migrations/20181002192054_fix_conversion_host_resource_type_spec.rb]: `update_attributes!`

**Comment by simaishi** [comment]
Backport approved by Engineering/QE.

Hammer backport details:

```
$ git log -1
commit f5006a3ab6f3e1e5f4820f074a75415ea4b37110
Author: Brandon Dunne <brandondunne@hotmail.com>
Date:   Tue Oct 2 23:44:40 2018 -0400

    Merge pull request #284 from agrare/fix_conversion_host_resource_type
    
    Fix invalid conversion host resource type
    
    (cherry picked from commit 2ce3c9a7bdf20ebe2541749e28d7d63eb92e9f71)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1634029
```

**Pull request merged**